core/src/main/scala/io/qbeast/core/model/QDataType.scala

```diff
@@ -52,29 +53,43 @@ trait OrderedDataType extends QDataType {
 
 object DoubleDataType extends OrderedDataType {
   override def name: String = "DoubleDataType"
+
   override val ordering: Numeric[Any] = implicitly[Numeric[Double]].asInstanceOf[Numeric[Any]]
 }
 
 object IntegerDataType extends OrderedDataType {
   override def name: String = "IntegerDataType"
+
   override val ordering: Numeric[Any] = implicitly[Numeric[Int]].asInstanceOf[Numeric[Any]]
 }
 
 object LongDataType extends OrderedDataType {
   override def name: String = "LongDataType"
+
   override val ordering: Numeric[Any] = implicitly[Numeric[Long]].asInstanceOf[Numeric[Any]]
 }
 
 object FloatDataType extends OrderedDataType {
   override def name: String = "FloatDataType"
+
   override val ordering: Numeric[Any] = implicitly[Numeric[Float]].asInstanceOf[Numeric[Any]]
 }
 
 object DecimalDataType extends OrderedDataType {
   override def name: String = "DecimalDataType"
+
   override val ordering: Numeric[Any] = implicitly[Numeric[Double]].asInstanceOf[Numeric[Any]]
 }
 
 object StringDataType extends QDataType {
   override def name: String = "StringDataType"
 }
+
+object TimestampDataType extends OrderedDataType {
```

Comment:
Since we are adding this in the core, we must also add a test in that package. You can give a look at `JSONSerializationTest`, `LinearTransformationTest` and `TransformerTest` to cover this.

Reply:
I added more tests for `JSONSerializationTest` and `TransformerTest`.